### PR TITLE
Updated multiple things

### PR DESCRIPTION
**Pull request by NicoJG**
I updated the following:  

- Suppress warnings generated by unicode-math that could lead to confusion for a few people. Also this makes the generated console output more clean.
- Removed the workaround for the siunitx micro symbol
- Updated the .gitignore, I mainly deleted `*.pdf` so that people can track their pdf plots.
- Replaced the included Eidesstattliche Versicherung with the official form provided by the TU.

There are still many things that are outdated, but all of those only concern the given example.pdf. 
For example, many URLs are outdated and lead to a 404 error page. 
The siunitx commands are outdate. `\unit`, `\qty` and `\ang` should be listed, not `\si` and `\SI`.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,4 +1,10 @@
 build/
 *.swp
-*.pdf
 
+# For Visual Studio Code files:
+.vscode/
```
Comment (maxnoe) [inline .gitignore:5]: Please don't include stuff related to the personal development environment in project gitignores.

This belongs into your personal, user-wide gitignore, see:
https://git-scm.com/docs/gitignore (e.g. on linux, `$HOME/.config/git/ignore`

File: .gitignore [hunk 2]
```diff
@@ -1,4 +1,10 @@
...
+
+# OS specific trash containers (MacOS, Windows, Linux):
+.DS_Store
```
Comment (maxnoe) [inline .gitignore:8]: Same a above

**Review by maxnoe — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (NicoJG) [reply .gitignore]: Removed.

File: .gitignore (hunk 2 above)
Comment (NicoJG) [reply .gitignore]: Removed.

**Comment by NicoJG** [comment]
Sorry, I have not yet used GitHub workflows. From the log, it seems like `pdfpages` is not installed.
Does this mean I have to add `pdfpages` to the `tex-packages.txt` file?

**Comment by maxnoe** [comment]
@NicoJG Yes, exactly, `tex-packages.txt` is used to install all needed  packages into a minimal texlive installation

**Comment by NicoJG** [comment]
Now it should work.

**Review by maxnoe — APPROVED** [review]

**Comment by maxnoe** [comment]
thx

**Pull request merged**